circle.yml

```diff
@@ -110,3 +110,9 @@ experimental:
     branches:
       only:
         - staging
+deployment:
+  storybook:
+    branch: staging
+    commands:
+      - 'echo "sync_assets: false" >> locals.yml'
+      - 'cd apps && npm run storybook:deploy'
```

Comment:
maybe? I don't think so. Need to figure out a good way to test that. Perhaps I should add this branch to the config and have it tested that way? We probably need to generate a token specifically for this repo.